packages/grafana-ui/src/components/TimeSeries/utils.ts

```diff
@@ -401,7 +401,9 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<{
 
       if (customConfig.fillBelowTo) {
         const fillBelowToField = frame.fields.find((f) => f.name === customConfig.fillBelowTo);
```

Comment:
ok, i'm a bit confused here.

the values we save in the config are the _display_ name, shouldn't this logic also use that?

https://github.com/grafana/grafana/blob/6e6863f10a78014f757d4565330d67039449a447/public/app/plugins/panel/timeseries/FillBelowToEditor.tsx#L13

Reply:
The values we save in the config are the name values returned from the datasource since the field overrides have not yet run at that point. This is why in `utils.ts` we should check the `fillBelowTo` property against the original name and not against the display name.